vectordatasource/transform.py

```diff
@@ -4879,6 +4997,107 @@ def _normalize_kr_netref(network, ref):
     return network, ref
 
 
+def _normalize_no_netref(network, ref):
+    prefix, number = _splitref(ref)
+
+    if prefix == 'Rv':
+        network = 'NO:Riksvei'
+        ref = number
+
+    elif prefix == 'Fv':
+        network = 'NO:Fylkesvei'
+        ref = number
+
+    elif prefix == 'E':
+        network = 'e-road'
+        ref = 'E ' + number.lstrip('0')
```

Comment:
Do we strip out the space after the E somewhere else? (The space shouldn't be present by the time it exports in shield_text.)

Reply:
Yes, when we use `_splitref`, we get back a `ref` that has been stripped of all leading and trailing white space: https://github.com/tilezen/vector-datasource/blob/ec7b47c91b69728631778ffe258651ebdb723df2/vectordatasource/transform.py#L4955